### ISSUE TITLE
Kabonesa Wu / Dhegdheer Interaction [Test Server]

### ISSUE THREAD

**Issue by LiteSpecter** [mtgred/netrunner#3057]
If you have 3 creds and a Dhegdheer installed, clicking Wu won't let you install Magnum Opus.

For what it's worth, this "Can't Install" interaction persists on the live server with things like Street Peddler install cards with Tech Trader credits. This may have to be worked around manually

**Comment by LiteSpecter** [comment]
Also, Magnum appears greyed out for some reason when installing off Wu. Is there a reason for this?

**Comment by FarCryFromHuman** [comment]
About the greyed out MOpus, there might be some conflict around NRDB card ids. The core set MOpus is rotated out, but the revised core is in. Rotated ids are greyed out when you play Rebirth. May or may not be related?

**Comment by nealterrell** [comment]
Someone else reported the grey card issue with Celebrity Gift. Sounds like a Core 2.0 thing. I'll pass it on to @danhut.

**Comment by nealterrell** [comment]
And yeah... the other issue has no timeline for fixing. It's currently too difficult to know all potential sources of credits/discounts when allowing a card to be installed.

**Comment by nealterrell** [comment]
You're off the hook, @danhut -- it's a server-side issue.

**Comment by danhut** [comment]
I assume we load cards from the original core on the server side in preference to the revised core?  When we merge we should drop the original core car right?

**Comment by nealterrell** [comment]
Yeah, it's just an ordering issue... right now it's using database order (which is arbitrary), so I'll have it sort by `:rotated` so those cards get overridden by the non-rotated versions.
